src/FormLayout/Generic.php

```diff
@@ -88,6 +88,38 @@ public function addGroup($label = null)
         return $this->add(new self($label));
     }
 
+
+    public function addSection($hasDivider = true)
+    {
+        $v = $this->add('View');
+        if ($hasDivider) {
+            $this->add(['ui' => 'hidden divider']);
```

Comment:
shouldn't divider go before View ?

Reply:
If not place after, then Submit button is placed right after field in last section.